modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,25 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. Optional: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
+Devfile::
+* If your Git repository contains a Devfile, it is automatically detected and populated on the *Devfile path* field.
+* To edit the file import type and select a different strategy, click *Edit import strategy* option.
+* If multiple Devfiles are detected, the Devfile path needs to be specified, relative to the *Context Dir* field, to import a specific Devfile.
+
+Dockerfile::
+* If your Git repository contains a Dockerfile, it is automatically detected and populated on the *Dockerfile path* field.
+* To edit the file import type and select a different strategy, click *Edit import strategy* option.
+* If multiple Dockerfiles are detected, specify the path of the specific Dockerfile relative to the *Context Dir* field.
+
+Devfile and Dockerfile::
+* If a Devfile and a Dockerfile are detected in the same repository, the Devfile is selected by default. To choose a different strategy, click *Edit import strategy*.
+* If multiple Devfiles and Dockerfiles are detected in the same repository, you can specify which file you wish to import.
+
+. After the URL is validated in the *Builder* section, it finds and selects an appropriate builder image and marks it with a star. For `https://github.com/sclorg/nodejs-ex` Git URL, the default Node.js builder image is selected.
+.. If the builder image is not auto-detected, select a builder image.
+.. Optional: Use *Builder Image Version* drop-down list to specify a version.
+.. Optional: Use *Edit import strategy* to select a different strategy.
```

Comment:
Suggest:
. After the URL is validated in the *Builder* section, it finds and selects an appropriate builder image and marks it with a star. If the builder image is not auto-detected, select a builder image. For `https://github.com/sclorg/nodejs-ex` Git URL, the default Node.js builder image is selected.
.. Optional: Use *Builder Image Version* drop-down list to specify a version.
.. Optional: Use *Edit import strategy* to select a different strategy.